src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -500,17 +500,9 @@ public virtual bool AllowAutoRedirect
             }
         }
 
-        public override string ConnectionGroupName
-        {
-            get
-            {
-                throw NotImplemented.ByDesignWithMessage(SR.net_PropertyNotImplementedException);
-            }
-            set
-            {
-                throw NotImplemented.ByDesignWithMessage(SR.net_PropertyNotImplementedException);
-            }
-        }
+        // .Net Framework behavior difference: ConnectionGroupName cannot be changed.
+        // For app-compat reasons we allow apps to change it.
```

Comment:
Nit: I don't think we need the comment.  The first line is essentially referring to what, after this is checked in, will have been an old implementation that never shipped and won't actually exist anywhere.  Same with the other occurrence of it below.